addons/web/static/src/js/views/kanban/kanban_column.js

```diff
@@ -29,7 +29,8 @@ var KanbanColumn = Widget.extend({
         'click .o_kanban_load_more': '_onLoadMore',
         'click .o_kanban_toggle_fold': '_onToggleFold',
```

Comment:
commit message not precise enough on purpose. Actually you cannot drag and drop anymore starting at the buttons quick add or settings  or somewhere in the menu settings itself. Maybe you should say that the drag and drop is only available starting over the title?